Fixes #148: Adds missing metadata properties to GTFS dataset entity creation

maximearmstrong opened the pull request:
**Summary:**

This PR adds the missing properties to the GTFS dataset entity creation.

Implement

- Adds Bounding box and octagon properties
- Adds Number of agencies, routes and stops properties

Changes made

- `create_dataset_entity_for_gtfs_metadata.py` was adapted to reflect changes
- `process_routes_count_by_type_for_gtfs_metadata.py` and `process_stops_count_by_type_for_gtfs_metadata.py` were adapted to reflect changes.
- `request_utils.py` was adapted

Mobility Database properties added ([displayed here](http://staging.mobilitydatabase.org/w/index.php?title=Special:ListProperties/&limit=50&offset=50))

- Bounding octagon (P68)
- Number of stops (P69)
- Number of stations (P70)
- Number of entrances (P71)
- Number of agencies (P72)
- Number of routes (P73)
- route type (P74)


**Expected behavior:** 

Adds the bounding box and octagon, Number of agencies, routes and stops to a Dataset entity during its creation.


Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Run the unit tests with `pytest` to make sure you didn't break anything
- [x] Format the title like "<short description of fix and changes>" (for example - "Check for null value before using field")
- [x] Linked all relevant issues
- [ ] ~Include screenshot(s) showing how this pull request works and fixes the issue(s)~

maximearmstrong commented:
@LeoFrachet @timMillet The following properties have been added to the database ([here](http://staging.mobilitydatabase.org/w/index.php?title=Special:ListProperties/&limit=50&offset=50)), and property codes have been added to the [data model document](https://docs.google.com/document/d/1wJ9f4MYPrrJJYmYEB7xpRIAJ9XaBzw8UaCUqg-q23HE/edit#).

Bounding octagon (P68)
Number of stops (P69)
Number of stations (P70)
Number of entrances (P71)
Number of agencies (P72)
Number of routes (P73)
route type (P74)

Also, timestamps ([here](http://staging.mobilitydatabase.org/wiki/Property:P66) and [here](http://staging.mobilitydatabase.org/wiki/Property:P67)) have been created as String instead of Point in time, because Wikibase won't display the timestamps under the ISO8601 format as desired, but rather as a date.

Full example is [displayed here](http://staging.mobilitydatabase.org/wiki/Item:Q224)

Could you confirm these new/modified properties are ok for the data model? Thanks

timMillet commented:
I checked the data model document. 2 things:
- Does the number of stops (P69) also include the stops with `location_type=blank`? If not, are these stops counted anywhere else?
- In the document, the definitions of number of route (P73) and route type (P74) are the same. Is that wanted?
Otherwise, all good for me!

maximearmstrong commented:
Good points, answers following:

> * Does the number of stops (P69) also include the stops with `location_type=blank`? If not, are these stops counted anywhere else?
P69 includes `location_type=blank`, so I adjusted the document. Also, I added that stations are `location_type=1` and entrances are `location_type=2`.

> * In the document, the definitions of number of route (P73) and route type (P74) are the same. Is that wanted?
 It is not! I fixed it :)

maximearmstrong commented:
@timMillet Good points, answers following:

> Does the number of stops (P69) also include the stops with `location_type=blank`? If not, are these stops counted anywhere else?

P69 includes `location_type=blank`, so I adjusted the document. Also, I added that stations are `location_type=1` and entrances are `location_type=2`.

> In the document, the definitions of number of route (P73) and route type (P74) are the same. Is that wanted?

 It is not! I fixed it :)
